composables/useFieldWithValidation.ts

```diff
@@ -35,6 +35,10 @@ interface UseFieldWithValidationReturns {
     e: Event | string | number | boolean | null,
     shouldValidate?: boolean
   ) => void;
+  /**
+   * Function which resets field validation
+   */
+  handleReset: () => void;
```

Comment:
IMHO it could be renamed to `handleResetValidation`, because maybe we would want later expose a function that resets the field value. What do you think ?

Reply:
Yep, I think that's a fair point 😄